src/main/java/com/conveyal/datatools/manager/models/FeedVersion.java

```diff
@@ -382,22 +414,6 @@ public void validate(MonitorableJob.Status status) {
 
             // This will persist the document to Mongo
             this.mobilityDataResult = Document.parse(json);
```

Comment:
@miles-grant-ibigroup We need to be careful here because we don't want to create a new Mongo object if the other validator is expecting to do that. See ValidateFeedJob -> jobFinished(). Both validators need to be aware of the Mongo object state, so that either can create or update. Let me take a look. Good spot btw.

Reply:
@miles-grant-ibigroup This should now be addressed. Unit tests look good. Lemme know how you get on.